sdk/eventhub/azure-eventhub/TROUBLESHOOTING.md

```diff
@@ -185,7 +248,9 @@ When filing GitHub issues, the following details are requested:
 <!-- repo links -->
```

Comment:
Can we add a link next to the logs bullet point above for the logging section in this guide?